docs/content/metric-stores/overview.md

```diff
@@ -0,0 +1,106 @@
+# Overview
+
+A MetricStore provides properties to set the metrics of a Feathub job into an
+external metric service. A metric refers to a statistic of a characteristic of a
+feature.
+
+## Metric store configurations
+
+Below are common configurations shared by different metric store
+implementations. The document of each metric store contains its specific
+configurations.
+
+| Key                 | Required | Default   | Type    | Description                                                  |
+| ------------------- | -------- | --------- | ------- | ------------------------------------------------------------ |
+| type                | Required | -         | String  | The type of the metric store to use.                         |
+| report_interval_sec | Required | -         | Integer | The interval in seconds to report metrics.                   |
```

Comment:
I have changed the type of this configuration to `Number` here. 

This configurations definition and the configuration validation process is also updated to support `Union[float, int]`. Please refer to the latest code for details.

Reply:
`Float` is simpler to understand than `Number`. For example, `jobmanager.memory.jvm-overhead.fraction` has type `Float` in Flink doc [1].

Can you explain why we should use `Union[float, int]` instead of simply `float`?

[1] https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/

Reply:
If the type of this configuration is `float` and users configure`report_interval_sec: 5`, an exception would be raised here.
https://github.com/alibaba/feathub/blob/master/python/feathub/common/config.py#L178